Index: server.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/server.js b/server.js
--- a/server.js	
+++ b/server.js	
@@ -1,0 +1,7 @@
+const app = require('./app');
+
+const PORT = 3000;
+
+app.listen(PORT, () => {
+    console.log(`Server started and runnung on port, ${PORT}`);
+});
\ No newline at end of file
Index: app.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app.js b/app.js
--- a/app.js	
+++ b/app.js	
@@ -1,0 +1,6 @@
+const express = require('express');
+
+
+const app = express;
+
+module.exports = app;
\ No newline at end of file
