Index: app.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>const express = require('express');\r\nconst morgan = require('morgan');\r\n\r\nconst tourRouter = require('./routes/tourRoutes');\r\nconst userRouter = require('./routes/userRoutes');\r\n\r\nconst app = express()\r\n\r\n/*\r\nMiddleware\r\n* **********************************************/\r\napp.use(morgan('dev'));\r\napp.use(express.json());\r\n\r\napp.use((req,res,next)=>{\r\n    console.log(`Hello from the middleware\uD83D\uDC4B\uD83D\uDC4B\uD83D\uDC4B\uD83D\uDC4B\uD83D\uDC4B\uD83D\uDC4B\uD83D\uDC4B\uD83D\uDC4B`);\r\n    next();\r\n})\r\napp.use((req,res,next)=>{\r\n    req.requestTime = new Date().toISOString();\r\n    next();\r\n})\r\n\r\n//Mounting Routes\r\napp.use('/api/v1/tours', tourRouter);\r\napp.use('/api/v1/users', userRouter);\r\n\r\nmodule.exports = app;
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app.js b/app.js
--- a/app.js	
+++ b/app.js	
@@ -1,3 +1,9 @@
+const express = require('express');
+
+
+const app = express;
+
+module.exports = app;
 const express = require('express');
 const morgan = require('morgan');
 
